scripts/run-cypress-test-docker

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+if bash +x -o nounset -c "$(aws ecr get-login --no-include-email --region "us-west-2" --registry-ids "${AWS_ACCOUNT_ID}")" ; then
+  docker-compose pull db_migrate easi
+
+  docker-compose build --parallel easilocal cypress
+
+  docker-compose up -d db
+  sleep 5
```

Comment:
This will fail intermittently. We should use `psql` to poll until the database is awake. It would look something like this:

```sh
echo -n "Waiting for database to come up"
until psql --command "SELECT version();" &>/dev/null ; do
  echo -n "."
  sleep 1
done
```

Reply:
The `run-cypress-test-docker` script is running in our primary truss circleci image, which doesn't have `psql` installed. I removed the sleep command and the tests are still passing. Since we're starting the docker-compose services incrementally, and we start db_migrate after this which uses the `FLYWAY_CONNECT_RETRIES` setting, this should hopefully be okay for now. I created https://jiraent.cms.gov/browse/EASI-632 to improve control of the start-up order.

Reply:
If `FLYWAY_CONNECT_RETRIES` is already play then that is absolutely the way to do the waiting and I agree with removing the `sleep` here.